Add sampling info to Tracing tasks (#3397)
istio/istio.io: content/boilerplates/trace-generation.md

> To see trace data, you must send requests to your service. The number of requests depends on Istio's sampling rate.

Reviewer: This boilerplate breaks the ordered lists in your procedures. Do not use a boilerplate for any text that is duplicated. There are several things you need to consider when creating boilerplate content. Please just include the content as steps in each of the procedures.

Author: I have considered the context around the text. These *aren't* separate steps. This is further explanation of the first step which is to send traffic. It belongs in the same step.

Nothing about this breaks the ordered list in any fashion.  Note the 1 followed by the 2 below. 

![image](https://user-images.githubusercontent.com/21148125/53750250-1b74d400-3e5e-11e9-89c4-349b0536bd5f.png)

> The number of requests you need to send in order to see trace data depends on the sampling rate set at the time Istio was installed.

Reviewer: Please avoid using boilerplates without considering the content around it. You are using numbered steps in the document but not in the boilerplate. Not only that, but the numbering of lists in the boilerplate won't sync with the numbering in the document if you do. Boilerplate needs to be used carefully and for fully independent content. 
In this case, I would add the steps to the flow in the appropriate place on each file as part of the respective numbered list.

Reviewer: The text needs to be changed to fit into the numbered steps of the procedure. be explicit about what the command does to avoid confusing readers.

```suggestion
1. To see trace data, you must send requests to your service. The number of requests depends on Istio's sampling rate. You set this rate when you install Istio. The default sampling rate is 1%. You need to send at least 100 requests before the first trace is visible. To send a 100 requests to the `productpage` service, use the following command:
```

Author: I explicitly did not number this boilerplate, as I have thoughts about using it elsewhere (in a FAQ) and because it is a sub-step of the larger list in which it sits.

I have adopted your wording.

> By default, the sampling rate is set to 1%, meaning that you need to send at least 100 requests before the first trace will be visible.

Reviewer: ```suggestion
```

Author: done.

> To send multiple requests, use a command like:

Reviewer: ```suggestion
```
